springwolf-plugins/springwolf-sns-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/SnsAsyncOperationBinding.java

```diff
@@ -20,4 +20,10 @@
 public @interface SnsAsyncOperationBinding {
```

Comment:
Both `protocol` and `endpoint` are mandatory fields for which I don't think we should provide a default value. This decision automatically generates a breaking change. Without a value, the compilation of any code using this annotation will fail.